proto_parser/README.md

```diff
@@ -0,0 +1,15 @@
+# Proto++
```

Comment:
I don't think you need readme for this lib.

You can add readme for the demo, like how to run the demo end to end.